docs/src/tutorials/qless.md

```diff
@@ -0,0 +1,160 @@
+```@example qless1
+# The Q-less QR factorization may be used to solve the least-norm problem
+#
+#   minimize ‖x‖  subject to  Ax=b
+#
+# while saving storage because Q is not formed.
+# Thus it is appropriate for large problems where storage is at a premium.
+# The normal equations of the second kind AAᵀy = b are the optimality conditions of the least-norm problems, where x = Aᵀy.
+# If Aᵀ = QR, they can be equivalently written RᵀRy = b.
+#
+# This procedure is backward stable if we perform one step of iterative refinement---see
+#
+# Å. Björck, Stability analysis of the method of seminormal equations for linear least squares problems,
+# Linear Algebra and its Applications, 88–89, pp. 31-48, 1987, DOI 10.1016/0024-3795(87)90101-7.
```

Comment:
Sorry, this needs to be adjusted. The method of seminormal equations for the least-norm problem is analyzed in by Paige in https://www.ams.org/journals/mcom/1973-27-122/S0025-5718-1973-0331745-1.

For normal equations, the stability with and without Q is similar.

But it's not the same for least squares. It's only for least squares that we need iterative refinement. I think it would be useful to write functions that solve those problems and hide the details.

Reply:
I will add iterative refinement in a separate PR.

Reply:
I'm saying the text needs to be adjusted too.